kuma/wiki/tests/test_views.py

```diff
@@ -1761,9 +1761,10 @@ def _run_translate_edit_tests(edit_slug, edit_data, edit_doc):
                 eq_(0, len(Document.objects.filter(title=edit_data['title'] +
                                                    ' Redirect 1',
                                                    locale=foreign_locale)))
-                self.assertRedirects(response, reverse('wiki.document',
-                                                       locale=foreign_locale,
-                                                       args=[edit_doc.slug]))
+                doc_url = reverse('wiki.document', args=[edit_doc.slug], locale=foreign_locale)
+                params = {'rev_saved': ''}
+                doc_url = '%s?%s' % (doc_url, urlencode(xparams))
```

Comment:
This should be ``params`` not ``xparams``, as caught by [flake8](https://travis-ci.org/mozilla/kuma/jobs/228551964#L382).  It's also code that isn't called during tests, but we agreed to postpone cleanup until after this PR is merged.